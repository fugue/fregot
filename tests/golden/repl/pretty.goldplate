{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":load pretty.rego",
        "people[name]",
        "rev_people[person]"
    ]
}
