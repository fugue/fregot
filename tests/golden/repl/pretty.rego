package pretty

people = {
  "Tom Nook": {"address": "Sun Isle 20", "age": 52}
}

rev_people[v] = k {
  people[k] = v
}
