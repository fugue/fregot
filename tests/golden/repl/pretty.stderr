F u g u e   R E G O   T o o l k i t
fregot v0.12.2 repl - use :help for usage info
Loading pretty.rego...
Loaded package pretty
